query-algebrizer/src/types.rs

```diff
@@ -35,6 +36,14 @@ pub enum DatomsTable {
     FulltextValues,     // The virtual table mapping IDs to strings.
     FulltextDatoms,     // The fulltext-datoms view.
     AllDatoms,          // Fulltext and non-fulltext datoms.
+    Computed(usize),    // A computed table, tracked elsewhere in the query.
+}
+
+/// A source of rows.
```

Comment:
Elaborate: `/// A source of rows produced dynamically by the underlying SQL store.` or similar.